src/cljs/webchange/game_changer/views.cljs

```diff
@@ -139,30 +71,24 @@
                                     (handle-next {:data     current-data
                                                   :callback #(reset! current-step next-idx)})
                                     (reset! current-step next-idx)))]
-
-    ;(print "current-step" @current-step)
-    ;(print "current-data" @current-data)
-
     (logger/trace-folded "Game changer data" @current-data)
-
     (let [current-step-data (get-current-step-data @current-data @current-step @steps)
           {:keys [component title timeline handle-next next-enabled? next-step-idx prev-step-idx]} current-step-data
-
           first-step? (= @current-step 0)
           last-step? (= @current-step (dec (count timeline)))
-
+          select-template-step? (= title "Choose Activity")
```

Comment:
This flag may break if we change step title. It would be better to use step id.

- add `:id` to step data in `.../game_changer/steps/select_template/index.cljs`
- receive if from `current-step-data` (do not forget to update `get-current-step-data` function for id getting)
- use `select-template-step? (= id :select-template)` expression